ci/travis/osx/before_install.sh

```diff
@@ -18,7 +18,8 @@ brew remove gdal || true
 
 brew tap osgeo/osgeo4mac
 brew update
-brew install osgeo/osgeo4mac/qgis-214 --without-postgresql --only-dependencies
+brew install grass-70
+brew install osgeo/osgeo4mac/qgis-214 --without-postgresql --only-dependencies --WITH-GRASS7
```

Comment:
The `--WITH-GRASS7` is not needed. I meant the CMake config would be enabled if `brew install grass-70` precedes `brew install osgeo/osgeo4mac/qgis-214` (which it does now).